engine/src/main/scala/cromwell/engine/db/slick/ExecutionInfoComponent.scala

```diff
@@ -45,6 +47,16 @@ trait ExecutionInfoComponent {
     } yield (execution, executionInfo)
   )
 
+  val runningExecutionsAndExecutionInfosByWorkflowId = Compiled(
+    (id: Rep[String]) => for {
+      executionInfo <- executionInfos
+      execution <- executionInfo.execution
+      workflowExecution <- execution.workflowExecution
+      if workflowExecution.workflowExecutionUuid === id
```

Comment:
Yeah seems the check for workflow status happens [all the way up in the WorkflowManagerActor](https://github.com/broadinstitute/cromwell/blob/cd4f74dd0c9fee9c0b91ecb9786082d8a7741833/engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala#L406).